samples/TextGeneration.swift

```diff
@@ -0,0 +1,90 @@
+// Copyright 2024 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import GoogleGenerativeAI
+import XCTest
+
+@available(iOS 15.0, macOS 11.0, macCatalyst 15.0, *)
+final class TextGeneration: XCTestCase {
+  override func setUpWithError() throws {
+    try XCTSkipIf(APIKey.default.isEmpty, "`\(APIKey.apiKeyEnvVar)` environment variable not set.")
+  }
+
+  func testTextOnlyPrompt() async throws {
+    // [START text_gen_text_only_prompt]
+    let generativeModel =
+      GenerativeModel(
+        // Specify a Gemini model appropriate for your use case
+        name: "gemini-1.5-flash",
+        // Access your API key from your on-demand resource .plist file (see "Set up your API key"
+        // above)
+        apiKey: APIKey.default
+      )
+
+    let prompt = "Write a story about a magic backpack."
+    let response = try await generativeModel.generateContent(prompt)
+    if let text = response.text {
```

Comment:
don't know if that's common in snippets in swift, but should you include an `else` for this case?

Reply:
So far we haven't included an `else` in any of our Swift snippets but it's something to consider. Something like:
```
else {
  print("No text in response.")
}
```
Alternatively we could `guard let`:
```
guard let text = response.text else {
  fatalError("No text in response.")
}
```
I'm a little nervous that the `fatalError`s will end up in production code if devs copy/paste and don't change anything but we already have multiple in cases like generating text from images.

Reply:
Absolutely, it's possible that anything in the samples end up inadvertently in final versions of the code. In the case of kotlin, since we `print` whatever comes back, if it's missing it will be printed as `null`, which makes the snippet complete, without having to fully address what to do if it's missing.

That reminds me that we should have error handling snippets somewhere...